doc-src/BackupPC.pod

```diff
@@ -1337,13 +1337,24 @@ The requirements for each Xfer Method are:
 
 To use rsync, you need rsync-bpc installed on the BackupPC server.
 
-On the client, you should have at least rsync 3.x.  Rsync is run on the remote
-client via ssh.
-
 The relevant configuration settings are $Conf{RsyncClientPath},
 $Conf{RsyncSshArgs}, $Conf{RsyncShareName}, $Conf{RsyncArgs},
 $Conf{RsyncArgsExtra}, $Conf{RsyncFullArgsExtra}, and $Conf{RsyncRestoreArgs}.
 
+On the client, you should have at least rsync 3.x.  Rsync is run on the remote
+client via ssh.
+
+On some new glibc versions (like in Ubuntu 20.10, Fedora 33beta) and rsync V3.2.3 there is a bug, which prevents to set the correct owner, group and permissions of restored files. You will find many of the following entries in rsync.log : 
+
+    rsync: [receiver] failed to set permissions on "/some_file": Operation not supported (95)
+    rsync: [receiver] failed to set permissions on "/some_file2": Operation not supported (95)
+    ...
+
+Solution :
+compile rsync Yourself on the backuppc clients, atm of this writing (2021-July-03) rsync version 3.2.4dev works fine.
```

Comment:
It is not clear what users should do.

- just rebuild rsync 3.2.3
- or they should take some actions (e.g. set some compile options or downgrade glibc) and rebuild rsync 3.2.3
- upgrade to rsync 3.2.4dev (or later) by any means (e.g. from a package if their distribution provides it)